doc/_dev-guide/chapter_0/section_3.md

```diff
@@ -60,12 +60,67 @@ ${HOMEBREW_PREFIX}/sbin/syslog-ng -Fdevt
 
 this will give you detailed information of what {{ site.product.short_name }} does.
 
+### Using MacPorts
+
+First, check [[this|dev-platform-build-macos#dependencies]] if you have not got MacPorts installed and pre-configured yet.
+
+Installation location of MacPorts will be referenced as `${MACPORTS_PREFIX}` in this document, as if you follow the installation instructions above it will be set already correctly independenty of your system.
```

Comment:
```suggestion
Installation location of MacPorts will be referenced as `${MACPORTS_PREFIX}` in this document, as if you follow the installation instructions above it will already be set correctly independent of your system.
```